login-fire-common-behavior.html

```diff
@@ -80,7 +80,7 @@
        */
       appName: {
         type: String,
-        value: ""
```

Comment:
All double quotes in JS code were replaced by single quote to follow [Google JavaScript style guide](https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes).